### PR TITLE
Run all tests for entire project

### PR DESCRIPTION
**Pull request by lawrencejones**
Aim to verify ruby and node versions are correct, then kick off all tests regardless of language.

@joelotter could you have a look please?

### CODE REVIEW

**Comment by lawrencejones** [comment]
Sample output...

screenshot 2016-01-15 15 15 00

File: bin/test [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+require_relative '../utils/diggit'
+
+def verify_version(program, expected, actual)
+  unless expected == actual
```
Comment (JoelOtter) [inline bin/test]: This will only work if the versions are exactly the same (I think; I don't speak Ruby). Do you not want a _minimum_ version?

Comment (lawrencejones) [inline bin/test]: Think I want exact version actually, given that it doesn't cost much to install the exact version and it gets rid of niggly inconsistency issues

**Comment by lawrencejones** [comment]
Think I may change this to a makefile sometime in the future but it will do for now. Thanks for the review!

**Pull request merged**